content/enterprise/admin/infrastructure/admin-cli.mdx

```diff
@@ -22,7 +18,9 @@ Note that `tfe-admin` is an alias for `replicated admin`, and can be used interc
 tfe-admin support-bundle
 ```
 
-This command generates a support bundle for all nodes. The support bundles will be uploaded to the same object store bucket that is used to store Terraform state files. The support bundles for a specific run of the admin command will all be uploaded to a directory with the same JobID (a timestamp in [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) format). If you are sending a support bundle to HashiCorp Support, package and send all associated bundles to ensure that we have all the necessary information.
+This command generates a support bundle for all nodes in `/var/lib/replicated/support-bundles`.
```

Comment:
This reads (to me) as though support bundles are generated for nodes that exist in that file path. Could we rephrase tobreak this up?

"This command generates a support bundle for all nodes.

For Standalone (single node) installations the support bundle will be created in `/var/lib/replicated/support-bundles`.